app/src/main/java/ro/code4/monitorizarevot/data/model/County.kt

```diff
@@ -21,17 +22,33 @@ class County {
     lateinit var name: String
 
     @Expose
+    @SerializedName("numberOfPollingStations")
```

Comment:
SQLite doesn't allow to change the column name with the ALTER TABLE command(to do this in a migration we would need to make a new table, copy the old data and then delete the old table) . To fix this I simply mapped the old limit field with the actual field returned by the server through GSON.